assets/js/src/common/subscribers_cache_message.tsx

```diff
@@ -6,14 +6,27 @@ type Props = {
   cacheCalculation: string;
 };
 
-const handleRecalculate = () => {
-  window.location.reload();
-};
-
-const SubscribersCacheMessage = ({ cacheCalculation }: Props): JSX.Element => {
+export function SubscribersCacheMessage({ cacheCalculation }: Props): JSX.Element {
   const [loading, setLoading] = useState(false);
   const datetimeDiff = new Date().getTime() - new Date(cacheCalculation).getTime();
   const minutes = Math.floor((datetimeDiff / 1000) / 60);
+
+  const handleRecalculate = () => {
+    setLoading(true);
+    MailPoet.Ajax.post({
+      api_version: MailPoet.apiVersion,
+      endpoint: 'settings',
+      action: 'recalculateSubscribersCountsCache',
+    }).done(() => {
+      window.location.reload();
+    }).fail((response) => {
+      MailPoet.Notice.error(
```

Comment:
I think we should stop using these old notices. There is the React version that should be used nowadays. We should probably deprecate this old one.